lib/matplotlib/figure.py

```diff
@@ -362,7 +362,7 @@ def _repr_html_(self):
         # We can't use "isinstance" here, because then we'd end up importing
         # webagg unconditiionally.
         if (self.canvas is not None and
-            'WebAgg' in self.canvas.__class__.__name__):
+                'WebAgg' in self.canvas.__class__.__name__):
```

Comment:
No, my point was "this should be something like `self.canvas.__class__.__name__ == "<something>"`" (or `self.canvas.__class__.__module__ == "<something>"`) rather than a substring check.